draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -1093,371 +1093,185 @@ Content-Type: text/plain
 </section>
 </section>
 
-<section title="Header Fields" anchor="header.fields">
+<section title="Header and Trailer Fields" anchor="header.fields">
+   <t>
+      HTTP messages use key/value pairs to convey data about the
```

Comment:
These paragraphs are okay, but I think it would work better to describe them commonly first, as fields within a potential header section or trailer section, and then "In general, we refer to these as HTTP message fields, or just "fields".

Note the "Every message has two ..." is a bit strong. HTTP messages have a (potentially empty) header section and potentially one or more trailer sections.